pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -5596,10 +5598,25 @@ static void pjsua_call_on_tsx_state_changed(pjsip_inv_session *inv,
 			    val_str.ptr = val + STR_DURATION.slen;
 			    val_str.slen = input.slen - STR_DURATION.slen;
 			    info.duration = pj_strtoul(&val_str);
+			} else {
+			    info.duration = PJSUA_UNKNOWN_DTMF_DURATION;
```

Comment:
Kind of forgot about this. This should have been a separate PR.

`info.duration` was left uninitialized if the duration was not specified in the message body, making the field basically useless (no way of knowing whether the field is valid or not).